packages/compass-sidebar/src/components/multiple-connections/sidebar.tsx

```diff
@@ -22,45 +32,139 @@ const sidebarStyles = css({
   height: '100%',
 });
 
-// TODO We will get rid of this placeholder when expose the necessary props outside
-// eslint-disable-next-line
-const noop_tmp = (() => {}) as any;
+type ConnectionErrorToastBodyProps = {
+  info: ConnectionInfo | null;
+  onReview: () => void;
+};
+
+const connectionErrorToastBodyStyles = css({
+  display: 'flex',
+  alignItems: 'start',
+  gap: spacing[2],
+});
 
+const connectionErrorToastActionMessageStyles = css({
+  marginTop: spacing[1],
+  flexGrow: 0,
+});
+
+function ConnectionErrorToastBody({
+  info,
+  onReview,
+}: ConnectionErrorToastBodyProps): React.ReactElement {
+  return (
+    <span className={connectionErrorToastBodyStyles}>
+      <span>
+        There was a problem connecting{' '}
+        {info ? `to ${getConnectionTitle(info)}` : ''}
+      </span>
+      {info && (
+        <Link
+          className={connectionErrorToastActionMessageStyles}
+          hideExternalIcon={true}
+          onClick={onReview}
+        >
+          REVIEW
+        </Link>
+      )}
+    </span>
+  );
+}
 // Having props here is useful as a placeholder and we will fix it with the first props.
 // eslint-disable-next-line
-export function MultipleConnectionSidebar({}: MultipleConnectionSidebarProps) {
+export function MultipleConnectionSidebar({
+  appName,
+}: MultipleConnectionSidebarProps) {
+  const { openToast, closeToast } = useToast('multiple-connection-status');
```

Comment:
Once again, just so we continue to be on the same page: this code doesn't belong in the sidebar and should be refactored out to a connection plugin at some point. If you are still planning to use only COMPASS-7397 as an overarching ticket for this work, it would be good to add a todo here

Reply:
I think I'll use this ticket for this change, as it should be small, instead of using COMPASS-7397.